using_images/other_images/jenkins.adoc

```diff
@@ -766,6 +766,47 @@ You can increase the amount of memory available to Jenkins by overriding
 the *MEMORY_LIMIT* paramenter when instantiating the Jenkins Ephemeral or
 Jenkins Persistent template.
 
+[[build-pod-retention]]
+=== Build Pod Retention
+
+Pipeline build pods are deleted by default after the build completes or is aborted. 
```

Comment:
I would clarify that these build pods are Jenkins agent/slave pods created by kubernetes plugin.